PrivateTalkApp/Home/View/HomeView.swift

```diff
@@ -7,9 +7,97 @@
 
 import SwiftUI
 
+// MARK: - Constants
+private struct Constants {
+    static let ADD_SCHEDULE_BUTTON_IMAGE_NAME = "plus.circle.fill"
+    static let TODAY_BUTTON_TEXT_KEY = LocalizedStringKey("today_button_text")
+    static let HEADER_AND_CALENDAR_SPACING = 8.0
+    static let ELEMENTS_IN_THE_HEADER_SPACING = 0.0
+    static let HEADER_TITLE_FONT_SIZE = 20.0
+    static let TODAY_BUTTON_AND_ADD_SCHEDULE_BUTTON_SPACING = 20.0
+    static let ADD_SCHEDULE_BUTTON_WIDTH = 30.0
+    static let ADD_SCHEDULE_BUTTON_HEIGHT = 30.0
+    static let HEADER_HORIZONTAL = 10.0
+    static let MAIN_STACK_PADDING = 5.0
+}
+
+// MARK: - ホーム View
 struct HomeView: View {
+    
+    // カレンダーのViewModel
+    @StateObject private var calendarViewModel = CalendarViewModel()
+    // 有効(true): 今日ボタン押せない ／ 無効(false): 今日ボタン押せる
+    @State private var todayButtonEnable: Bool = true
+    
+    var body: some View {
+        VStack(spacing: Constants.HEADER_AND_CALENDAR_SPACING) {
+            // ヘッダー
+            headerView
+            // カレンダー
+            CalendarView(calendarViewModel: calendarViewModel,
+                         todayButtonEnable: $todayButtonEnable) { date in
+                self.calendarViewModel.setDisplayDate(date: date)
+            }
+        }
+        .padding(.vertical, Constants.MAIN_STACK_PADDING)
+    }
+    
+    // ヘッダー部分
+    var headerView: some View {
+        HStack(spacing: Constants.ELEMENTS_IN_THE_HEADER_SPACING) {
+            // 年月テキスト
+            Text(calendarViewModel.calendarModel?.displayYearMonthString ?? String.empty)
+                .font(.system(size: Constants.HEADER_TITLE_FONT_SIZE))
+                .fontWeight(.bold)
+                .foregroundColor(.foreground)
```

Comment:
[foregroundColor](https://developer.apple.com/documentation/swiftui/view/foregroundcolor(_:))はdeprecated
```suggestion
                . foregroundStyle(.foreground)
```

Reply:
Fixed.